test/base64_unittest.cc

```diff
@@ -0,0 +1,38 @@
+#include "../src/base64.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+TEST(EncodeTest, EmptyEncode) {
+  const std::string empty_str("");
```

Comment:
May not be worth naming the constants... How about just inlining them into the EXPECT_EQ? I.e.,
```cpp
TEST (EncodeTest, EmptyEncode) {
  EXPECT_EQ("", base64::Encode(""));
}
```
?

Reply:
Agreed - done.